docs/sources/tutorials/api_org_token_howto.md

```diff
@@ -9,13 +9,14 @@ weight = 10
 
 # API Tutorial: How To Create API Tokens And Dashboards For A Specific Organization
 
-A common scenario is to want to via the Grafana API setup new Grafana organizations or to add dynamically generated dashboards to an existing organization.
+Use the Grafana API to setup new Grafana organizations or to add dynamically generated dashboards to an existing organization.
 
 ## Authentication
 
-There are two ways to authenticate against the API: basic authentication and API Tokens.
+There are two authentication methods to access the API: 
 
-Some parts of the API are only available through basic authentication and these parts of the API usually require that the user is a Grafana Admin. But all organization actions are accessed via an API Token. An API Token is tied to an organization and can be used to create dashboards etc but only for that organization.
+- Basic authentication: A Grafana Admin user can access some parts of the Graphana API through basic authentication. 
```

Comment:
"Graphana" -> "Grafana"

Reply:
Looks good, Marcus!

I was browsing through the Grafana website and the graphs on the Home page just occupied my mind!